UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/appsettings.local.overrides.json

```diff
@@ -30,7 +30,8 @@
       "StorageContainerName": "ess-fulfilment",
       "DynamicQueueName": "ess-{0}-fulfilment",
       "LargeExchangeSetSizeInMB": 300,
-      "SmallExchangeSetSizeInMB": 50
+      "SmallExchangeSetSizeInMB": 50,
+      "ExchangeSetTypes": "sxs, mxs, lxs"
     }
```

Comment:
This is inconsistent with appsettings.json. I assume the spacing after the comma is not significant?

"ExchangeSetTypes": "sxs, mxs, lxs"

Reply:
Fixed, Spacing is not significant after comma it must be without space.